### DISCUSSION TITLE
关于六分街的Random Play

### DISCUSSION THREAD

**Discussion by GeZiCat2499** [share121/inter-knot#1484] [其他]
![image](https://github.com/user-attachments/assets/06a5cc72-ad3c-4199-8893-6a0abdceaf77)

上次逛街去了六分街那边，有幸发现了个看起来很不错的录像带店。然而架子上录像带太多了，店员推广的又都是星徽骑士一类的老片，望有熟悉该店的大佬推荐一下。
（求恐怖片大佬放过）

**Comment by Jiucheng258** [comment]
《第一杯土》就挺不错的，我还挺喜欢看的来着。

**Comment by ZhuYuan-148** [comment]
上次和店长看了《真相不止一个》，还不错的侦探喜剧，就是主角的头发有点…奇特？

**Comment by bin-lin2** [comment]
我想看《十万次心跳》

**Comment by WORLD-executeME** [comment]
我比较喜欢《以太绮梦》和《十万次心跳》
